uk.co.compucorp.civicrm.hrcore/backstop_data/casper_scripts/login.js

```diff
@@ -1,11 +1,23 @@
 'use strict';
 
-module.exports = function (casper) {
+module.exports = function (casper, scenario) {
   var config = require('../site-config');
   var loginFormSelector = 'form#user-login-form';
+  var credentials = config.credentials[scenario.credential || 'admin'];
 
   casper
-    .echo('Logging in before starting...', 'INFO')
+    .then(function () {
+      if (casper.exists('a[href="/user/logout"]')) {
+        casper.echo('Current scenario has different login credentials from previous, logging out is necessary', 'INFO');
+        casper.echo('Logging Out', 'INFO');
+        return casper.evaluate(function () {
```

Comment:
what if you simply do `phantom.clearCookies();` see here http://darrendev.blogspot.sk/2014/03/clearing-cookies-between-casperjs-tests.html

Reply:
Yes it works, thanks for the suggestion. So I will take the ticket out of POC and estimate.